proto/dymensionxyz/dymension/eibc/events.proto

```diff
@@ -12,24 +12,22 @@ option go_package = "github.com/dymensionxyz/dymension/v3/x/eibc/types";
 message EventDemandOrderCreated {
   // order_id is the unique identifier of the demand order.
   string order_id = 1;
-  // fulfillment_id is the unique identifier of the fulfillment.
+  // price is the price of the demand order.
   string price = 2;
-  // fulfillment_amount is the amount of the fulfillment.
+  // fee is the fee of the demand order.
   string fee = 3;
-  // is_fulfilled is the flag indicating whether the order is fulfilled.
-  bool is_fulfilled = 4;
```

Comment:
why did we remove it? old clients don't need it?
also shouldn't we deprecate it? seems like we've changed it.